js/stex.js

```diff
@@ -152,26 +152,31 @@ module.exports = class stex extends Exchange {
 
     async fetchTicker (symbol, params = {}) {
         await this.loadMarkets ();
-        let market = this.market (symbol);
-        let response = await this.publicGetTicker (this.extend (market['id'], params));
-        return this.parseTicker (response, market);
+        let response = await this.fetchTickers (undefined, params); // Cannot find a public method of retriving specific ticker on STEX, fetching all instead.
+        let ids = Object.keys (response);
+        for (let i = 0; i < ids.length; i++) {
+            let id = ids[i];
+            if (id === symbol) {
+                return response[id];
```

Comment:
In order to keep it unified we need to make sure the ticker is properly parsed here.

- https://github.com/ccxt/ccxt/wiki/Manual#price-tickers

Example: 

- https://github.com/ccxt/ccxt/blob/master/js/hitbtc2.js#L721
- https://github.com/ccxt/ccxt/blob/master/js/hitbtc2.js#L777
- https://github.com/ccxt/ccxt/blob/master/js/hitbtc2.js#L790

Reply:
Hmm, on parseTicker?

The output have identical layout to hitbtc2: https://testnet.powermining.pw/ccxtccxtpull4549.html (comparasion between hitbtc2, stex and graviex - fetchTicker('LTC/BTC))

It have already been parsed through fetchTickers, so i don`t really see a reason it have to go through parseTicker another time? Maybe there is something i'm not seeing?